app/styles/common.scss

```diff
@@ -47,17 +44,7 @@
   @extend .btn-bold;
 }
 .btn-subtle {
-  background: #f4f4f4;
-  @extend .mediumgray;
-  border-color: #ccc;
-}
-.btn-subtle:hover,
-.btn-subtle:focus,
-.btn-subtle.focus,
-.btn-subtle:active,
-.btn-subtle.active,
-.open > .dropdown-toggle.btn-subtle {
-  background: #e2e2e2;
+  @include buttons.button-variant(bootstrap.$btn-default-color, #f4f4f4, #ccc);
```

Comment:
We use the `button-variant` Bootstrap mixing to generate the styles for subtle buttons instead of doing them manually. That gives us support for disabled buttons for free, which we didn't have before.